src/vs/base/browser/dom.ts

```diff
@@ -1087,13 +1075,7 @@ function _$<T extends Element>(namespace: Namespace, description: string, attrs?
 		}
 	});
 
-	for (const child of children) {
-		if (child instanceof Node) {
-			result.appendChild(child);
-		} else if (typeof child === 'string') {
-			result.appendChild(document.createTextNode(child));
-		}
-	}
+	result.append(...children);
```

Comment:
Will this implicitly create text nodes for strings?

Reply:
Yes.